packages/framework-core/package.json

```diff
@@ -55,6 +55,7 @@
     "mock-jwks": "^0.3.1",
     "nock": "^13.0.4",
     "sinon": "9.2.3",
-    "sinon-chai": "3.5.0"
+    "sinon-chai": "3.5.0",
+    "metadata-booster": "0.3.1"
```

Comment:
I believe this package should live together with the framework? Also it should have the `@boostercloud` package namespace

Reply:
I would prefer it not to live in the framework repository, as this can be used by anyone in any project (I mean the transformer code)

However, I agree with you that we can add the @boostercloud namespace. Let's see if I can change that